x/poe/contract/tg4_engagement.go

```diff
@@ -77,7 +77,7 @@ func NewEngagementContractAdapter(contractAddr sdk.AccAddress, twasmKeeper types
 	}
 }
 
-func (a EngagementContractAdapter) UpdateAdmin(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error {
+func (a EngagementContractAdapter) UpdateBootstrapAccountAddress(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error {
```

Comment:
this is not correct. We still have an "admin" role within the PoE contract. Please revert this change

Reply:
updated